app/src/main/java/fr/free/nrw/commons/mwapi/ApacheHttpClientMediaWikiApi.java

```diff
@@ -54,6 +50,20 @@ public ApacheHttpClientMediaWikiApi(String apiURL) {
         api = new CustomMwApi(apiURL, httpClient);
     }
 
+    /**
+     * File entity id obtained after uploading the image
+     *
+     * @param fileName title of the image uploaded
+     */
+    @Override
+    public String getFileEntityId(String fileName) throws IOException {
+        return "M" + api.action("query")
```

Comment:
Explanation `// Create Wikibase media id from the page id. Example media id: M80618155 for https://commons.wikimedia.org/wiki/File:Tantanmen.jpeg with has the pageid 80618155`

Reply:
done